src/tls/Makefile.OCaml

```diff
@@ -51,10 +53,6 @@ ifeq (,$(wildcard $(MLCRYPTO_HOME)/CoreCrypto.cmxa))
   $(error I cannot find $(MLCRYPTO_HOME)/CoreCrypto.cmxa -- please run make in $(MLCRYPTO_HOME))
 endif
 
-ifeq (,$(wildcard $(FFI_HOME)/FFICallbacks.cmxa))
-  $(error I cannot find $(FFI_HOME)/FFICallbacks.cmxa) -- please run make in $(FFI_HOME))
-endif
-
```

Comment:
why remove this?

Reply:
I tried to set up general project-wide dependencies in Makefile.common, but the CI doesn't seem to like it yet.

Reply:
ok, it should be brought back; I'll add a note in https://github.com/mitls/mitls-papers/wiki/The-build-system-of-Everest to explain why